app.py

```diff
@@ -238,9 +243,9 @@ def conversation_without_data(request):
         return jsonify(response_obj), 200
     else:
         if request.method == "POST":
-            return Response(stream_without_data(response), mimetype='text/event-stream')
+            return Response(stream_without_data(response))
```

Comment:
"text/event-stream" is the mime-type for SSE. However, we are returning a streaming HTTP response, which is a superset of SSE. We shouldn't set this mime-type as it confuses the browser Network panel. (The fetch request seems alright with it, so it's been working fine in the UI)